downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -53,8 +53,78 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.4.0](https://git
 
 ### New features
 
+#### Ability to get the strand dump during `bal test`
+When running the tests in a Ballerina package or a file using the `bal test` command, the strand dump can be obtained by sending the `SIGTRAP` signal to that process denoted by the `BTestMain` class-name.
```

Comment:
```suggestion
When running the tests in a Ballerina package or a file using the `bal test` command, the strand dump can be obtained by sending the `SIGTRAP` signal to that process.
```
I think we can remove the java related details because I can do `bal test --native` and send the `SIGTRAP` signal to the native process and get the strand dump.